storages/merger/list.go

```diff
@@ -0,0 +1,134 @@
+package merger
+
+import (
+	"bytes"
+	"encoding/xml"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"sort"
+	"strconv"
+
+	"github.com/allegro/akubra/log"
+	"github.com/allegro/akubra/storages/merger/s3datatypes"
+	"github.com/allegro/akubra/transport"
+)
+
+// MergeListResponses unifies responses from multiple backends
+func MergeListResponses(successes []transport.ResErrTuple) (resp *http.Response, err error) {
```

Comment:
I would include `Bucket` in method's name, so `MergeListResponses`, because now it suggests, that it can merge any type of responses, that return a list.